sass/_legacy-docs.scss

```diff
@@ -0,0 +1,13 @@
+.legacy-doc-warning {
+    padding: 20px;
```

Comment:
I think we should prefer `rem`. `1rem` is 16px with default font size IIRC so this could be something between `1rem` and `1.5rem`.